Camera2SlowMotion/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -22,27 +22,11 @@
 
     <fragment
         android:id="@+id/permissions_fragment"
-        android:name="com.example.android.camera2.slowmo.fragments.PermissionsFragment"
+        android:name="com.example.android.camera2.slowmo.fragments.PermissionsAndCameraSettingFragment"
         android:label="Permissions" >
-
-        <action
-            android:id="@+id/action_permissions_to_selector"
-            app:destination="@id/selector_fragment"
-            app:popUpTo="@id/permissions_fragment"
-            app:popUpToInclusive="true" />
-
-    </fragment>
-
-    <fragment
-        android:id="@+id/selector_fragment"
-        android:name="com.example.android.camera2.slowmo.fragments.SelectorFragment"
-        android:label="Selector" >
-
         <action
-            android:id="@+id/action_selector_to_camera"
-            app:launchSingleTop="true"
-            app:destination="@id/camera_fragment" />
-
+          android:id="@+id/action_permissions_fragment_to_camera_fragment"
```

Comment:
probably don't need the "fragment" here, the others didn't use it before.

Reply:
The name suggest the link between `permission_fragment` and `camera_fragment`. There was another UI panel in this diagram before which I removed viz. `selector_fragment`.
Thus, this new action replaces previous links from `permission_fragment` -> `selector_fragment`  -> `camera_fragment` with `permission_fragment` -> `camera_fragment`
![Screenshot 2020-08-05 at 9 00 04 AM](https://user-images.githubusercontent.com/23431811/89368831-14f52680-d6fa-11ea-9a4b-b875905f060f.png)